File: src/python/itkwasm/itkwasm/float_types.py
from enum import Enum

class FloatTypes(str, Enum):
    Float32 = 'float32'
    Float64 = 'float64'
    SpacePrecisionType = 'float64'

    def __str__(self):
        return str(self.value)

    def __repr__(self):
        return f'"{str(self.value)}"'

File: packages/core/java/src/main/java/org/itk/wasm/FloatTypes.java
package org.itk.wasm;

public enum FloatTypes {
    Float32("float32"),
    Float64("float64"),
    SpacePrecisionType("float64");

    private final String value;

    FloatTypes(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}
